README.md

```diff
@@ -1,80 +1,64 @@
-* Explore – Technology we’re interested in but need to evaluate before recommending as the default choice. Items in this category should have a team willing to sponsor the exploration.
-* Endure – We’ve used this technology in the past, but it’s no longer recommended.
-* Retire - This technology is holding us back and should be actively removed. Items in this category should have a desired removal date.
+Items on the Tech Radar should only be captured if the cost of change is high or the benefits of standardizing outweigh are for the good of Redgate or our customers.
+Each technology choice is positioned in one of four areas.
+* Adopt – We feel strongly that Redgate should be adopting these items. It’s the default choice.
+* Trial – Worth pursuing. It’s important to understand how to build up this capability. Redgate teams can try this technology on a project that can support the risk.
+* Assess – Worth exploring with the goal of understanding how it will affect Redgate
+* Hold - Proceed with caution
 
-The radar only applies to all actively developed products at Redgate. 
+The items in the Tech Radar apply to all products in the Growth category. The Tech Radar doesn’t provide all the answers, but it should give strong direction in those areas where choice makes a difference. For example, should we be favouring containers over Virtual Machines? What technology (Hyper-V vs VMWare)? Are we adopting TypeScript?
 
-Items in the [primary Tech Radar](https://radar.thoughtworks.com/?sheetId=https%3A%2F%2Fraw.githubusercontent.com%2Fred-gate%2FTech-Radar%2Fmain%2Fradar.csv) have an expectation that it should be followed (where appropriate) for all teams at Redgate. For example, if C# X arrives then we get most benefit from adopting it wholesale across the organization, whereas a particular date/time library is a local choice for a development team.
+We're following Thoughtworks approach. For discussion on the quadrants, please see the [Thoughtworks Radar Faq](https://www.thoughtworks.com/radar/faq)
 
-## Libraries
+## What isn’t on the Tech Radar?
+We only use the Tech Radar to capture decisions relevant to our growth products (those that we are investing in for the future). We don’t use the Tech Radar to capture historical decisions that are no longer relevant. For example, we’d prefer to say the technology we use going forward rather than listing all failed attempts in the past.
 
-The [Library Tech Radar](https://radar.thoughtworks.com/?sheetId=https%3A%2F%2Fraw.githubusercontent.com%2Fred-gate%2FTech-Radar%2Fmain%2Fradar_libraries.csv) aims to support teams with local decisions by sharing experience from other teams.
+Examples of things that probably SHOULD NOT be on the tech radar
+* RavenDB – it’s not a choice anyone is going to hit in the future
+* Inno Setup – Backup isn’t a growth product
+* Agile techniques – again, we don’t believe in one correct way and capturing a list of techniques that do/do not work isn’t worth it
```

Comment:
Given that @Greg-Smulko has raised concern around removing the libraries tech radar, it's worth capturing in here _why_ that's the case if we truly believe it is the right decision.

For the record, I _do_ believe it's the right decision. The choice of low-level libraries shouldn't be a division wide concern IMO. Where it matters (e.g, due to shared libraries where dependencies can cause some pain) then I'd argue the solution there is for discussion to open between the provider and clients.

There's no consensus of which libraries open source software is allowed to use to reduce friction and yet we pull in a multitude of open source nuget packages. If the open source world can solve it for an unknown cardinality of consumers, then we should be able to figure it out for our distinctly smaller set of consumers.

Reply:
I see your point Chris, but I still think that from the pragmatic point of view it would be useful to have some kind of a blueprint/guidelines for new products (and also for existing products that are being modernized) - see https://github.com/red-gate/Tech-Radar/pull/342#discussion_r578368618.

Reply:
Is a full tech radar overkill for that? Perhaps just a list of libraries would suffice.

Reply:
I think there's a place for capturing "don't use library X, prefer Y, for reasons" so that we don't lose knowledge gained the hard way... but not really in a radar. That feels more like an ADR thing, in which case perhaps the radar should explicitly say "if you don't find the answer you want here, also check all relevant ADRs"?

Reply:
See https://github.com/red-gate/Tech-Radar/pull/342#discussion_r578368618 . 
This was moved to https://info.red-gate.com/display/PD/Recommended+libraries. Resolving.